DataConnectors/Connector_syslog_WatchGuardFirebox.json

```diff
@@ -0,0 +1,117 @@
+{
+    "id": "WatchguardFirebox",
+    "title": "Firebox",
+    "publisher": "Watchguard",

```

Comment:
provide should match the company name in cases:

WatchGuard or WatchGuard Technologies